t5x/scripts_gpu/README.md

```diff
@@ -0,0 +1,79 @@
+# GPU Scripts
```

Comment:
`scripts_gpu` should go inside contrib/gpu

Reply:
What do you think about the top-level Dockerfile? Does that need to be moved as well?

Reply:
Yes let's move it inside  the contrib becuase it is only used there

Reply:
Done